tools/c7n_azure/c7n_azure/resources/resource_map.py

```diff
@@ -36,6 +36,7 @@
     "azure.front-door": "c7n_azure.resources.front_door.FrontDoor",
     "azure.front-door-policy": "c7n_azure.resources.front_door_policy.FrontDoorPolicy",
     "azure.hdinsight": "c7n_azure.resources.hdinsight.Hdinsight",
+    "azure.host-pools": "c7n_azure.resources.host_pools.HostPools",
```

Comment:
general preference is for singular resource names, the type name represents the class.